src/Pages/ContactPage.js

```diff
@@ -21,11 +25,55 @@ class ContactPage extends React.Component {
     return (
       <div>
         <NavBar />
-        <Container className="contactUs">
-          <h1>Have any questions?</h1>
-          <h2>Send us an email</h2>
-          <ContactForm />
-        </Container>
+        <div class="bg-dark"></div>
+        {/******WORD******/}
+        <div className="contactWordLocation">
+          <container className="contactWord">
+            <h1 class="display-3">Have any questions ?</h1>
+            <br></br>
+            <h2 class="display-5">we are happy to help <span class="display-1">:)</span></h2>
+
+
+            <br></br>
```

Comment:
Please refrain from using `<br />` for spacing. This would cause the mobile view to look weird. Use CSS component `margin-top` or `margin-bottom` instead. (Note: use `vw` or `vh` as your unit in CSS. ) 
![Screen Shot 2022-06-18 at 5 29 43 PM](https://user-images.githubusercontent.com/49298813/174457563-b1e06363-6579-46f9-8132-33a3f15011ae.png)